src/variables/sortBy.ts

```diff
@@ -0,0 +1,4 @@
+export const variables = {
+  sort_by_alphabetically: 'alphabetically',
+  sort_by_length: 'length',
+};
```

Comment:
```suggestion
export const SORT_BY = {
  alphabetically: 'alphabetically',
  length: 'length',
  none: 'none',
};
```
suggest improve naming. and make value for initial state instead empty string.